test/libsolidity/semanticTests/tryCatch/create.sol

```diff
@@ -6,19 +6,19 @@ contract Succeeds {
 }
 
 contract C {
-    function f() public returns (Reverts x, uint, string memory txt) {
```

Comment:
Not sure what the `uint` was for. It's unused so it does not make any difference other than adding an extra zero in expectations. It does seems too relevant to the test so I removed it.